backend/qemu.pm

```diff
@@ -872,7 +872,14 @@ sub start_qemu ($self) {
         }
 
         sp('device', 'usb-kbd') if $use_usb_kbd;
-        sp('smp', $vars->{QEMUTHREADS} ? [qv "$vars->{QEMUCPUS} threads=$vars->{QEMUTHREADS}"] : $vars->{QEMUCPUS});
+
+        my $smp_config = [$vars->{QEMUCPUS}];
+        for my $key (qw(QEMUSOCKETS QEMUDIES QEMUCLUSTERS QEMUCORES QEMUTHREADS)) {
+            my $qkey = lc(substr $key, 4);
```

Comment:
```suggestion
            my $qkey = lc(key =~ s/QEMU//r);
```

to make it clear that the qemu prefix should be stripped regardless of length

Reply:
In that case I suggest to add `^`: `s/^QEMU//r` (I know it's not needed here but it says more explicitly what it does and it might be a nanosexond faster)

Reply:
Fixed.